contracts/abstracted_account.algo.ts

```diff
@@ -130,11 +163,17 @@ export class AbstractedAccount extends Contract {
    * Add an app to the list of approved plugins
    *
    * @param app The app to add
+   * @param global Whether or not this plugin should be approved for all addresses
    */
-  addPlugin(app: Application): void {
+  addPlugin(app: Application, global: boolean): void {
```

Comment:
I think it would make more sense to just pass the address here, with zeroAddress interpreted as global

Reply:
like `addPlugin(app: Application, address: Address. end uint64)`?

Reply:
Yep exactly